src/containers/Dashboard.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import AddEvent from '../components/AddEvent';
+import Events from '../components/Events';
+
+export default class Dashboard extends Component {
+    render() {
+        return (
+            <div>
+                <AddEvent />
+                <Events />
+            </div>
+        )
+    }
+}
```

Comment:
I think this should be moved inside the Views directory. Same goes for the Login and Register.

Reply:
fixed